apps/mosaico/static/hbl/abi.html

```diff
@@ -0,0 +1,40 @@
+<div class="static-page__abi">
```

Comment:
I think this file is missing Prettier formatting.

Reply:
Now It's prettier